controller.go

```diff
@@ -547,6 +563,11 @@ func (c *controller) Sandboxes() []Sandbox {
 
 	list := make([]Sandbox, 0, len(c.sandboxes))
 	for _, s := range c.sandboxes {
+		// Hide stub sandboxes from libnetwork users
+		if s.isStub {
+			continue
+		}
```

Comment:
@mavenugo I think it does break that particular scenario. We probably need a sandbox cleanup method in sandbox interface to cleanup left over sandbox. Will add that as part of this PR and will make necessary changes in docker when vendoring in.